File: Problem Solving in Java (Part-1)/src/AntonAndPolyhedrons.java
import java.util.Scanner;

public class AntonAndPolyhedrons {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int t = sc.nextInt();
        int sum = 0;
        for(int i = 0; i < t; i++) {
            String s = sc.next();

            if(s.equals("Tetrahedron")) {
                sum+=4;
            }
            else if(s.equals("Cube")) {
                sum+=6;
            }
            else if(s.equals("Octahedron")) {
                sum+=8;
            }
            else if(s.equals("Dodecahedron")) {
                sum+=12;
            }
            else if(s.equals("Icosahedron")) {
                sum+=20;
            }
        }
        System.out.println(sum);
    }
}

File: Problem Solving in C++ (Part-1)/AntonAndPolyhedrons.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;
    string word;

    int sum = 0;
    for(int i = 0; i < n; i++) {
        cin >> word;
        if(word == "Tetrahedron") {
            sum+=4;
        }
        else if(word == "Cube") {
            sum+=6;
        }
        else if(word == "Octahedron") {
            sum+=8;
        }
        else if(word == "Dodecahedron") {
            sum+=12;
        }
        else {
            sum+=20;
        }
    }
    
    cout << sum << endl;

    return 0;
}
